x-pack/plugins/reporting/server/core.ts

```diff
@@ -350,6 +338,36 @@ export class ReportingCore {
     return startDeps.esClient;
   }
 
+  public getScreenshots(options: ScreenshotOptions): Rx.Observable<ScreenshotResult> {
+    return Rx.defer(() => this.getPluginStartDeps()).pipe(
+      pluck('screenshotting'),
```

Comment:
I think it will be better to use `map` instead of `pluck`. [The docs for RxJS v7 show that `pluck` is deprecated](https://rxjs.dev/api/operators/pluck)